package.json

```diff
@@ -47,7 +47,7 @@
     "typescript": "^4.5.4"
   },
   "dependencies": {
-    "@sveltejs/vite-plugin-svelte": "^1.0.0-next.49",
+    "@sveltejs/vite-plugin-svelte": "^1.0.0",
```

Comment:
I didn't catch this in https://github.com/storybookjs/builder-vite/pull/428/files, but I think this needs to move to the svelte example, right?

Reply:
I don't think it can. `builder-vite` uses `loadSvelteConfig` and so it needs `vite-plugin-svelte`

It sounds like it will become a non-issue with Storybook 7. If there were a need to clean it up before then we could use a bundler like Rollup and make it a devDependency so that it will just extract out `loadSvelteConfig` and bundle it with the project so that users don't end up downloading the full `vite-plugin-svelte`

Reply:
Oh right, gotcha.  Then it should move into the dependencies of `packages/builder-vite`, not the top level of the monorepo.